packages/dlp/test/gapic_v2beta1.js

```diff
@@ -0,0 +1,341 @@
+/*
```

Comment:
ok. I will make it consistent between videointelligence and dlp by using hyphen instead